changelog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## Version 3.1.3 [Unreleased]
+
+### Changed
+- RenewTemprary URL Refactored to new service and now handles all the files
```

Comment:
RenewTemprary URL method refactored to new service and now handles all the files at once or something better.